content/en/docs/tutorials/kubernetes-basics/public/css/styles.css

```
@@ -8574,7 +8534,7 @@ button.close
 
 .nav
 {
-    font-family: Roboto Slab, Roboto, 'Helvetica Neue', Helvetica, 'Open Sans', Arial, sans-serif;
+    font-family: 'Helvetica Neue', Helvetica, 'Open Sans', Arial, sans-serif;
```

Can we try: 

```suggestion
    font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Here + below? 

Note: I'm not sure if this will actually work, but it's worth a shot – at the moment the tutorials are now falling back to the default/system font stack, for me:`-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif`.

Not a dealbreaker if we can't do this, but I'd like to try!